Demo/README.md

```diff
@@ -1,12 +1,12 @@
 # Demo
 
-Sample project to try out iOS SDK.
+Sample project to try out iOS Drop-in and Components integrations.
 
 ![DropIn preview](Screenshots/dropin-ios.jpg)
 
 ## How to start
 
-To start testing SDK, you need to provide the following information to `Configuration.swift`
+To start testing SDK, you need to open `Adyen.xcodeproj` in Xcode and provide the following information to `Demo/Configuration.swift`
```

Comment:
This section sounds like required configuration for running the Demo. It doesn't say anything about how to run it. How about renaming to something like "Configuration"?

Reply:
Renamed to "Configuration"